cmd/jaas/cmd/addcloudtocontroller.go

```diff
@@ -27,16 +27,15 @@ import (
 
 const (
 	addCloudToControllerCommandDoc = `
-The add-cloud-to-controller command adds the specified cloud to a specific 
-controller on jimm.
+Adds the specified cloud to a specific controller on jimm.
```

Comment:
suggestin: we should use uniform spelling of jimm.. either "JIMM" or "jimm", but it needs to be consistent in help text, at least.

Reply:
Yip, I'll change it to `JIMM`.